src/js/background.js

```diff
@@ -26,6 +27,17 @@ function showSafeNotification(packageName, packageVersion = null) {
   });
 }
 
+
+
+// Adding listeners for client side navs
+chrome.webNavigation.onHistoryStateUpdated.addListener(() => {
```

Comment:
We shouldn't need this as we're not doing any client side navigation on snyk.io. Did you encounter a situation where you needed this?

Reply:
Yes, switching the dependency tab options to show only vulnerable or all doesn't trigger the normal mechanism.